src/assets/icons/back-arrow-icon/back-arrow-icon.tsx

```diff
@@ -0,0 +1,19 @@
+import { Component, h, Prop } from '@stencil/core';
+import classNames from 'classnames';
+
+@Component({
+  tag: 'back-arrow-icon',
+  styleUrl: 'back-arrow-icon.scss',
+  shadow: true,
+})
+export class BackArrowIcon {
+  @Prop() class?: string;
```

Comment:
No need

Reply:
This is necessary because we have a default class of the element and it also receives an external class. 🙏

Reply:
Removed. 👍